### PR TITLE
nmblookup: add page

### PR DESCRIPTION
**Pull request by xeruf**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

There are probably more uses, this is just what I used so far :)

### CODE REVIEW

File: pages/linux/nmblookup.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# nmblookup
+
+> Discover SMB shares.
+
```
Comment (kbdharun) [inline pages/linux/nmblookup.md:4]:
```suggestion
> More information: <https://www.samba.org/samba/docs/current/man-html/nmblookup.1.html>.

```
Thanks for your contribution and welcome to tldr. I would suggest adding a link to the docs/website. 
I am not a user of this command so I will leave the review to others. Can you also sign the CLA?

File: pages/linux/nmblookup.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- Find hosts in the local network with SMB shares:
+
+`nmblookup -S '**'`
+
+- Find hosts in the local network with SMB shares ran by SAMBA:
```
Comment (kbdharun) [inline pages/linux/nmblookup.md:9]:
```suggestion
- Find hosts in the local network with SMB shares run by SAMBA:
```

**Review by pixelcmtd — COMMENTED** [review]
Please read the contributing guide before opening a PR.
Also: It's `samba`, why is this in `linux`?

**Comment by xeruf** [comment]
I thought samba is a linux tool, but since I don't use windows I don't know ^^

File: pages/linux/nmblookup.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# nmblookup
+
+> Discover SMB shares.
+
+- Find hosts in the local network with SMB shares:
+
+`nmblookup -S '**'`
```
[the remark this answers is not in the document]
Comment (xeruf) [reply pages/linux/nmblookup.md:8]: I would keep the short version in this example, to exemplify both

[the remark this answers is not in the document]
Comment (Waples) [reply pages/linux/nmblookup.md]: In our examples we use the long versions for better explanations.

**Comment by xeruf** [comment]
ready from my side :)

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**